### ISSUE TITLE
need upgrade: Accessibility.AXPropertyName allow value uninteresting

### ISSUE THREAD

**Issue by xueyc1f** [chromedp/cdproto#37]
[Accessibility.AXPropertyName](https://chromedevtools.github.io/devtools-protocol/tot/Accessibility/#type-AXPropertyName) Allowed Value: uninteresting ...
but [L258C1-L301C2](https://github.com/chromedp/cdproto/blob/master/accessibility/types.go#L258C1-L301C2) not has that
on call `nodes, err := accessibility.GetFullAXTree().Do(ctx)` got error: `json: cannot unmarshal JSON string into Go accessibility.PropertyName within "/nodes/1/ignoredReasons/0/name": unknown PropertyName value: uninteresting`
